CONTRIBUTING.md

```diff
@@ -57,6 +57,16 @@ python3 -m http.server 8080
 
 If you want to help maintain translations in the future, add yourself to `.github/DOCS_ISSUE_TEMPLATE.md`.
 
+### Style suggestions
+
+We are grateful for whatever help we can get. Here are a few tips:
+- We use docsfiy for document rendering. See that syntax for ways to include code fragments.
+- We prefer includes to code files rather than generating new snippets and including. (Some old examples make snippets, which we are slowly migrating.)
```

Comment:
```suggestion
- We prefer to reference real files in GitHub rather than creating snippets. (Some old examples make use of snippets but we are slowly migrating away.)
```